cmd/orb.go

```diff
@@ -732,7 +733,11 @@ func validateOrb(opts orbOptions, org orbOrgOptions) error {
 		return fmt.Errorf("failed to get the appropriate org-id: %s", err.Error())
 	}
 
-	_, err = api.OrbQuery(opts.cl, opts.args[0], orgId)
+	client := orb.GetClient(opts.cfg)
+	if client == nil {
+		return fmt.Errorf("Unable to validate orb")
```

Comment:
This error message obscures the underlying problem. Even though we may want to hide the inner workings from users, this could be too generic when the time comes to answer issues on GitHub.